### ISSUE TITLE
# 【選択課題】スマホのながら歩き対策(竹下翔太)

### ISSUE THREAD

**Issue by shotaaaaa** [gsc-aoyama/pokemongo4agu#75]
# 問題点
もともと日本では歩きスマホによる事故が多発しており、また今回のpokemon goも歩きながら遊ぶゲームのためより一層歩きスマホをする人の割合が増えてしまうと心配する声が多々挙げられている。実際海外では子供が危険な区域に侵入してしまったり、ポケモンを探して道路にでてしまうなどという事件が起こってしまっている。そのためこの問題はpokemon goを安全に楽しむためにも深く考えなくてはならない。
# 解決方法
ポケモンを探すときにスマホを見ながらでなくてはポケモンが見つからないようにするのではなく、ポケットに入れていてもポケモンが出現しているかどうかわかるようなシステムを作るべきだと思う。一つの案としてはポケモンがでたときは画面をOFFにしていてもバイブレーションが作動する、といったシステムなどを採用すれば、多少は改善されると思う。もしそれでもあまりにも改善されないようだったら、罰金などを設けるのも一つの案だと思う。

**Comment by mapconcierge** [comment]
すでに指摘の対策はされています。他に独自の提案がないと、このままでは評価が厳しいです。独創的なスマホのながら歩き対策を期待しています！
http://vitaminw.seesaa.net/article/440322252.html

**Comment by shotaaaaa** [comment]
了解です対策を追加します。
ポケモンgo上のマップに車や歩行者が多い時の通知、またどの方向からやってくるかを教えてくれるような機能をつけることで注意喚起をする。プレイヤーもこれらの注意を受けていくうちに少しずつ周りに注意をするようになるのではと思う。

**Comment by mapconcierge** [comment]
確かに、 Pokemon GOユーザーが増えると、逆に Pokemon GOユーザー同士の位置関係から、「２時の方向よりトレーナー接近中」とか、音声ナビしてくれるといいかもね。面白い！！
